tests/interface/error/driver.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2015, Oracle and/or its affiliates. All rights reserved
```

Comment:
Is this license header appropriate? The code shares little with the older interface tests.